### ISSUE TITLE
Add more quotes.

### ISSUE THREAD

**Issue by mubaris** [mubaris/motivate#3]

**Comment by harryjas** [comment]
@mubaris i am working on this

**Comment by mubaris** [comment]
Minimum of 20 :smile:

**Comment by jmeline** [comment]
I've opened a PR with some quotes for you. Let me know if I need to restructure them.

**Comment by mubaris** [comment]
If you are adding more quotes, add 20 quotes in a new json file in `data` folder. Then make a PR

**Comment by hydeenoble** [comment]
can more quotes still be added?. I would like to add more.

**Comment by mubaris** [comment]
Yeah. Sure.

**Comment by hydeenoble** [comment]
Great. on it

**Comment by harryjas** [comment]
okay..will iadd in a new json file..

**Comment by AlthafHussainK** [comment]
I would like to contribute to this project :hand:

**Comment by mubaris** [comment]
Best way to get started is to add more quotes. Fork the repo and add new json file containing 20 quotes. Then send a PR. :smile:

**Comment by AlthafHussainK** [comment]
I've come up with more quotes. Please have a look at it. :smile:

**Comment by dhanya1** [comment]
@mubaris I would like to contribute to this.

**Comment by Curchel** [comment]
i would like to contribute

**Comment by EveChen** [comment]
This project is awesome! I would like to contribute to this (even though I'm in the beginner level lol). 
My first step: Play with it (installing now).
2nd step:  Find 20 quotes and figure out how to make a json file (I know I should put it into the *data* folder :) )  
3rd step: PR - pull a request

I haven't contributed a project on Github before. Although I've already read articles related to "how to contribute to open source projects", you guys can correct me if I do something wrong. >"<

**Comment by hydeenoble** [comment]
@EveChen you are good to go.... Awesome!

**Comment by JayWelborn** [comment]
I added some quotes from famous artists about what motivated them to get work done.

**Comment by mubaris** [comment]
@dhanya1 @Curchel Everyone is welcome to contribute. :smile:

**Comment by fercreek** [comment]
I'm on it too!

**Comment by linsogles** [comment]
Go us!

**Comment by mubaris** [comment]
You're always welcome to add more quotes.

**Comment by Pr00bit** [comment]
hi there, I would like to also participate in this :). How I can do it?

**Comment by hydeenoble** [comment]
@Pr00bit you can create a new JSON file following the naming format and add a minimum of 20 quotes... :)

**Comment by Pr00bit** [comment]
hi thanks for explanation. Just would have small question - in which location the Json file needs to be upladed/created?

**Comment by hydeenoble** [comment]
OK.....I think there are many dynamics to this. However, when i contributed, I create a JSON file in the "data" folder which is in the "motivate" folder.

**Comment by Pr00bit** [comment]
Thanks for hint, will try to create Json file then :)

**Comment by Bubblemelon** [comment]
Hi @mubaris , I added 20 more quotes to the data folder and followed this [article on how to contribute](https://opensource.guide/how-to-contribute/#how-to-submit-a-contribution). Let me know if this is okay :)

**Comment by Jai-Z** [comment]
Hello, Can I add to this?

**Comment by abhinuvpitale** [comment]
You can use this scrapper to extract more quotes from good reads - [https://github.com/abhinuvpitale/Spiders/blob/master/goodreads_quotes_spider.py](

**Comment by thesanjeevsharma** [comment]
How about automating it to collect quotes by itself? @mubaris

**Comment by mubaris** [comment]
If you have a solution, Please share and make a PR

**Comment by Jai-Z** [comment]
Please check my PR, maintainer. Thanks for the opportunity.

**Comment by MegaWale** [comment]
i will add more quotes too #first_timer

**Comment by jervis446** [comment]
@mubaris can more motivate quotes still be added?. 
I would like to add more.

**Comment by toivomattila** [comment]
Hi, I added more quotes and just created my first pull request. Thank you for the opportunity and hopefully it's all good. :)

**Comment by AbhishekVarghese** [comment]
Hi, this project seems great. I am a first timer and was looking for a place to start. I went through the community chat here and this community seems great. Ill try to add a few more quotes. Glad to be a part of this !

**Comment by AbhishekVarghese** [comment]
Hey!

I just got an idea. We can create a screenlet/widget of this app and run it during startup. Every 15 min new quotes come up. So it sticks around the desktop in a corner. A constant Motivation ;)

How's the idea ? Should I work on this ?

**Comment by mubaris** [comment]
@AbhishekVarghese Your idea sounds awesome. If you can work on that, I'll be so glad. If you need any help, just ping me.

**Comment by AbhishekVarghese** [comment]
Sent a PR #82 
This is my first ever PR ;)
Thanks for letting me work on this :)

**Comment by Nythiennzo** [comment]
Please have look at #88 
It's my first time commenting on an issue.

**Comment by Kiailandi** [comment]
Hi guys, just a quick question / tip.

I'm working on a similar project (https://github.com/Kiailandi/quote_generator), a simple website to display quotes.

I'm using https://forismatic.com/en/api/ as a source for quotes and I was wondering if you'd like to use it as well or if you prefer to stick with hand picked quotes.

**Comment by mubaris** [comment]
**Instructions for OpenCode '18 Participants**

You can add multiple JSON files, but you get points only once

**Comment by AbhiRepository** [comment]
I would like to add more quotes! is it still open to contribution?

**Comment by mubaris** [comment]
@AbhiRepository Yep

**Comment by GOVINDDIXIT** [comment]
Can I add more quotes

**Comment by mubaris** [comment]
@GOVINDDIXIT You're welcome to add more quotes, but no more points will be awarded

**Comment by GOVINDDIXIT** [comment]
OK, No Problem

**Comment by gilitz** [comment]
Im a new contributer, can i add more quotes?

**Comment by mubaris** [comment]
Yep

**Comment by BenjaminGHo** [comment]
Hi!  I am also a new contributor, can I add in new quotes as well? :)

**Comment by mubaris** [comment]
Yep

**Comment by aashutoshrathi** [comment]
Can I add more quotes?

**Comment by hydeenoble** [comment]
Sure! @aashutoshrathi

**Comment by hrafnhildurs** [comment]
I put in a PR with more quotes if you want to accept it :)
https://github.com/mubaris/motivate/pull/136  #136

**Comment by Luv2C0de** [comment]
Hello @mubaris  may I add some quotes?

**Comment by Luv2C0de** [comment]
Hey @mubaris I added a PR #142

**Comment by AjaxianK** [comment]
Hi @mubaris , I want to contribute also, working on it ...

**Comment by JCalebBR** [comment]
Hey @mubaris I'd like to contribute as well

**Comment by harvipan** [comment]
Hey @mubaris , I would like to contribute to the project with 20 more quotes.

**Comment by rgh1002** [comment]
Hello, I'd like to contribute 45 quotes if you still need them. Thank you!

**Comment by lahorton** [comment]
Hey-
Just created a PR with a new json file of quotes.  It's my first PR, so please let me know if you have any suggestions!  Thanks!

**Comment by aananya27** [comment]
Hi, I am very new at this, but would like to contribute!
I guessed the best way to get started is to play around with it first, then fork-> add a new JSON with 20 or so quotes-> ask for PR
hope it works 
please let me know if I am on a wrong track 🤞 😬

**Comment by Gravity-I-Pull-You-Down** [comment]
@mubaris I want to contribute too.

**Comment by TheBatmanofButler** [comment]
Hi I would like to contribute too!

**Comment by rolisanchez** [comment]
Any plan to display quotes in other languages?

**Comment by TheBatmanofButler** [comment]
Just made a pull request! My first one, please let me know if there are any issues/mistakes :)

**Comment by yudina-m** [comment]
Hi! I've added 20 new quotes in the PR #210 
Great project!

**Comment by Nayan-Das** [comment]
I want to add quotes to this project.But it is saying permission denied.Could you help please.

**Comment by annied93** [comment]
I would like to start on this as well - I forked the repo. Should I try adding the quotes now?

**Comment by Deb23** [comment]
I would like to contribute to this. Why are we manually adding quotes? We can randomize it. Just a thought :)

**Comment by thronecloud** [comment]
Interested. Agreed with Deb23, would be good to have an API provide quotes on the fly. Either it can scrap off quotes from motivational sites or use some existing quotes API for the same

**Comment by ghost** [comment]
I would like to contribute!

**Comment by IsharaMadhavi** [comment]
This is an interesting project.Can i join to contribute?

**Comment by krlakshmikanth** [comment]
I would like to contribute to this project. Why don't we consume API that provides motivational quotes in real-time rather than adding and maintaining a manual inventory? Checkout some API link https://rapidapi.com/HealThruWords/api/universal-inspirational-quotes

**Comment by thronecloud** [comment]
Sure Lakshmi let's do it

**Comment by esilvaju** [comment]
Lets do it !

**Comment by realnicolasmoy** [comment]
Hi, I'd like to contribute to this repo as well.

**Comment by AdityaSPatil** [comment]
Added 20 new comments. @mubaris

**Comment by sQUARys** [comment]
Added 20 quotes #337

**Comment by enigmaticsoulrg** [comment]
Greetings, I'd like to contribute to this repo (beginner level). This is uplifting and motivating.

**Comment by raksh07** [comment]
I would like to contribute to this

**Comment by emmaalecrim** [comment]
Hi, I'm also contributing! Hooray! Gonna make my first PR.

**Comment by laurendc** [comment]
Hello, adding my contribution here! #384

**Comment by jacksonraniel** [comment]
Hi, everyone! I just did my first PR here.  #393

**Comment by chrisfoose** [comment]
I want to do this!

**Comment by avats-dev** [comment]
Is this still an active issue?

**Comment by slmnsh** [comment]
what if we setup a scrapper bot to fetch the quotes from a well know quote site??? in that way we can have so many quotes

**Comment by hydeenoble** [comment]
@salmannotkhan That sounds interesting.

**Comment by kpanic** [comment]
Added one more for Dennis Ritchie (DMR) https://github.com/mubaris/motivate/pull/411
RIP

**Comment by kpanic** [comment]
@hydeenoble @salmannotkhan 
There is some support for that:

For example:

```python
$ python -c 'from quotes_api import ForismaticAPI; import json; print(json.dumps(ForismaticAPI().get_random_quote(), ensure_ascii=False))' motivate/quotes_api.py
# Output manually reformatted, because github does not wrap long lines ;)
{
  'quote': 'Time stays long enough for anyone who will use it.',
  'author': 'Leonardo da Vinci'
}
```

**Comment by adya-13** [comment]
Hi, I would like to contribute to it.

**Comment by MVAMULYA** [comment]
Hi, I would like to contribute to it

**Comment by adya-13** [comment]
Hi, this is my first PR here. #419

**Comment by avats-dev** [comment]
### Hey everyone !! 

As, this project is not actively maintained now, I've forked it and updated it for the **contributors** [here](https://github.com/avats-dev/motivate). This is the concerned [**issue**](https://github.com/avats-dev/motivate/issues/1).
**PRs are welcome :tada:.**

There is also a new [gitter community](https://gitter.im/motivate-new/community) room for chat in case anyone needs help or to communicate with each other. :+1: 
[![Gitter](https://img.shields.io/gitter/room/avats-dev/motivate?label=Community%20Chat&logo=Gitter&style=for-the-badge)](https://gitter.im/motivate-new/community)

---

#### There are some of you who've already opened PR here, for them:
Just open a new PR with base `avats-dev/motivate` like below:
![change base](https://user-images.githubusercontent.com/35889327/91647244-1524df80-ea76-11ea-9eb3-8f5204dd92dc.gif)

---

#### For them who are new:
- fork [repo](https://github.com/avats-dev/motivate)
- clone https://github.com/avats-dev/motivate.git
- follow [these guides](https://github.com/avats-dev/motivate#contribution).

---
 ### This new [repo](https://github.com/avats-dev/motivate) is active, and I'm maintaining this. Feel free to drop a message on gitter chat if you're stuck or need help. 

> I would add instructions for **FIRST TIMERS** in a short while to assist them and get started with **OPEN SOURCE**. 

Thanks everyone for contributions. :heart:
